### PR TITLE
Bad 2 Da Bone [Skeleton Tweaks]

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
retools skeletons to be less weak and more of an antag-adjacent thing.
restores the broken skull behavior but not the rebuilding thing (as it was lame and buggy)
makes a few ghost roles for use in events.

credit to frigid and @Just-a-Unity-Dev for motorbike
**Screenshots**

**Changelog**

:cl:
- fix: Dying as a skeleton now makes you a skull again.
- tweaks: Skeletons are now much more robust.

### CODE REVIEW

File: Resources/Prototypes/Datasets/Names/skeleton_first.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - Phanages
+  - Pelvis
+  - Femur
+  - Tibia
+  - Fibula
+  - Patella
+  - Tarsalls
+  - Patella
+  - Tailbone
+  - Boner
```
Comment (moonheart08) [inline Resources/Prototypes/Datasets/Names/skeleton_first.yml:24]: boner johnson

**Review by moonheart08 — APPROVED** [review]

File: Content.Server/Mind/Components/TransferMindOnGibComponent.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using Content.Shared.Tag;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Mind.Components;
+
+[RegisterComponent]
+public sealed class TransferMindOnGibComponent : Component
```
Comment (mirrorcult) [inline Content.Server/Mind/Components/TransferMindOnGibComponent.cs:7]: i swear this alreaday exists/existed

File: Resources/Prototypes/Entities/Mobs/Species/skeleton.yml [hunk 3]
```diff
@@ -10,30 +10,55 @@
...
   - type: Damageable
     damageContainer: Biological
     damageModifierSet: Skeleton
+  - type: Barotrauma #no space damage.
+    damage:
+      types:
+        Blunt: 0
+  - type: Bloodstream
+    maxBleedAmount: 0
+    bloodReagent: Milk
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Mobs/Species/skeleton.yml:26]: SO based

File: Resources/Prototypes/Entities/Objects/Vehicles/buckleable.yml [hunk 4]
```diff
@@ -316,3 +316,53 @@
             path: /Audio/Effects/Vehicle/vehiclestartup.ogg
             params:
               volume: -3
+
+- type: entity
+  parent: BaseVehicle
+  id: VehicleSkeletonMotorcycle
+  name: Skeleton Motorcycle
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Vehicles/buckleable.yml:323]:
```suggestion
  name: skeleton motorcycle
```

File: Resources/Prototypes/Entities/Objects/Vehicles/buckleable.yml [hunk 5]
```diff
@@ -316,3 +316,53 @@
...
+      mask: /Textures/Effects/LightMasks/cone.png
+      autoRot: true
+    - type: ItemSlots
+      slots:
+        key_slot:
+          name: Keys
+          whitelist:
+            requireAll: true
+            tags:
+              - VehicleKey
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Vehicles/buckleable.yml:364]: any key? shouldnt it use its own key / not ATV stuff

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**